railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
@spastorino ok, I kind of like the idea. Then can we also add a feature like implicit serialization? I want to say `render @resource, serializer: ...` without explicitly specifying anything like `:json` or `:xml` since I wouldn't be able to take advantage of being format-agnostic unless you would need to return multiple formats. Also it should be possible to change AMS adapter on the fly. I can imagine people would want to use one serializer but return JSON data (`JSON` and `JSON_API`) in a different structure, depending on the request for transition purposes (e.g. iOS apps).

I have a feeling that you are assuming that new apps would return JSON data that follows the JSON API conversions. I understand http://jsonapi.org is just an example, but it seems like the new version of AMS is deeply tied into one specification. Are there any other specs out there already? If yes, AMS should be generalized so others can easily integrate with it. Right now I only see `JSON` and `JSON_API` adapters. If there was only a limited number of choices or just one, the benefit of being format-agnostic would be quite low.